Automattic/wp-calypso
# Help Center: Add current page url to search query mapping

Related to pdm0RZ-L3-p2

## Proposed Changes

Created a new hook that will map the current page URL to a search query. This will give users relevant search results in the Help Center whenever they navigate to a page that is in the mapping JSON file.


## Testing Instructions



* Checkout branch.
* Sandbox public API and test site.
* Navigate to `cd apps/editing-toolkit` and run `yarn dev --sync`.
* Load Calypso, open the help center, and navigate through all the links to test the different search query mappings.
* Ensure mappings are working for /wp-admin based pages (if specified in JSON).
* Ensure mappings are working in gutenberg editor (if specified in JSON).

## Pre-merge Checklist
